include/linux/soundwire/sdw.h

```diff
@@ -426,10 +426,9 @@ int sdw_slave_read_prop(struct sdw_slave *slave);
  * struct sdw_slave_id - Slave ID
  * @mfg_id: MIPI Manufacturer ID
  * @part_id: Device Part ID
- * @class_id: MIPI Class ID, unused now.
- * Currently a placeholder in MIPI SoundWire Spec
+ * @class_id: MIPI Class ID (defined starting with SoundWire 1.2 spec)
  * @unique_id: Device unique ID
- * @sdw_version: SDW version implemented
+ * @version: SDW version implemented
```

Comment:
Is this on purpose? We still use sdw_version anywhere else.

Reply:
something's wrong with GitHub. this is an old version that should no longer be there.

Reply:
fixed now, not sure what happened